### PR TITLE
Cars

### PR DESCRIPTION
**Pull request by John-athan14**

### CODE REVIEW

File: src/Components/VéhiculeCard/index.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+// import useState from 'react';
+import { SVéhiculeCard, Btn, Title, Image } from './style';
+
+export default function VéhiculeCard({ model, image, price }) {
+  // const [toggle, setToggle] = useState(false);
+  // const handleClick = () => {
```
Comment (Caroline-CD) [inline src/Components/VéhiculeCard/index.jsx]: Please remove all comments.

File: src/Components/VéhiculeCard/index.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+// import useState from 'react';
```
Comment (Caroline-CD) [inline src/Components/VéhiculeCard/index.jsx]: Please remove comments

File: src/Pages/Cars/index.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import VéhiculeCard from '../../Components/VéhiculeCard';
+import SCarsView from './style';
+
+function Véhicules() {
+  const [cars, setCars] = useState([]);
+
+  useEffect(() => {
+    axios.get('http://localhost:3031/cars').then(({ data }) => {
+      setCars(data);
+      console.log(data);
```
Comment (Caroline-CD) [inline src/Pages/Cars/index.jsx]: Please, remove console.log

File: src/Pages/CarsDetails/index.jsx [hunk 4]
```diff
@@ -0,0 +1,161 @@
+import Clio from '../../Assets/clio.png';
+// import Clio2 from '../../Assets/clio2.png';
```
Comment (Caroline-CD) [inline src/Pages/CarsDetails/index.jsx:2]: Please remove comments

File: src/Pages/CarsDetails/index.jsx [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+          </h2>
+
+          <p>Cette citadine présente depuis les</p>
+          <p>années 2006. Elle a su faire sa</p>
+          <p> place dans le marché français.</p>
+          <p>Présente en hybride, elle peut</p>
+          <p>maintenant être accessible en</p>
+          <p> voiture éléctrique grâce à notre</p>
+          <p>rétrofit.</p>
+          {/* <div className='Picture2'>
```
Comment (Caroline-CD) [inline src/Pages/CarsDetails/index.jsx:56]: Please remove comments

File: src/Pages/CarsDetails/index.jsx [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+              350km
+            </div>
+          </Contour>
+        </div>
+      </Container>
+      <ConfigurationOfCars>
+        <div>
+          <h2>Configuration du véhicule</h2>
+        </div>
+        {/* <div>
```
Comment (Caroline-CD) [inline src/Pages/CarsDetails/index.jsx]: Please remove comments

File: src/Pages/CarsDetails/style.jsx [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+`;
+
+const Container = styled.div`
+  height: 718px;
+  width: 1160px;
+  margin: auto;
+  background: #00e6c8;
+  border-radius: 20px;
+  /* display: flex;
+  flex-direction: column;
```
Comment (Caroline-CD) [inline src/Pages/CarsDetails/style.jsx]: please, remove comments

**Review by Caroline-CD — CHANGES_REQUESTED** [review]
Please can you remove comments :)

File: src/Components/VéhiculeCard/index.jsx (hunk 1 above)
Comment (John-athan14) [reply src/Components/VéhiculeCard/index.jsx]: comments remove

**Comment by John-athan14** [comment]
Remove comments code

**Review by Caroline-CD — APPROVED** [review]
Ok for me

**Review by Franck59840 — APPROVED** [review]

**Pull request merged**